ports/templates/ports/port-detail/summary.html

```diff
@@ -0,0 +1,114 @@
+{% load format_names %}
+
+<h3>{{ port.portdir }} | {{ port.name }}</h3>
```

Comment:
I just have a small feedback. In the tabbed view, when I switch from the summary tab to some other tab (builds or tickets or installs) I do not see for what port the information is being shown. IMO it'd be better to have the port name above those 4 tabs. Example:

<img width="1440" alt="Screen Shot 2019-06-15 at 12 36 09 AM" src="https://user-images.githubusercontent.com/9131544/59532336-005a3b80-8f06-11e9-90b4-45dbb93d8a8b.png">

Reply:
Thank you, I have done the changes: also mentioned here: https://github.com/macports/macports-webapp/issues/7#issuecomment-502347279

Reply:
Yes, this was done and I'm happy with the result.